### DISCUSSION TITLE
Unsure how to create aliases/variables

### DISCUSSION THREAD

**Discussion by HarmonBlues** [autokey/autokey#618] [Q&A] [answered]
I want to rebind wasd for a game that doesn't allow me to rebind keys to esdf.

I'm unsure what the nomenclature is to create vars.

in Autohotkey I used to be able to do something like:
e::w
return

**Comment by josephj11** [comment, marked as answer]
Try the following for one of the keys. If it works as desired, repeat for the others.

Create a phrase with a trigger abbreviation of w set to trigger immediately and as part of a word. You may or may not want to ignore the case of the typed abbreviation. Set the phrase text to the new letter you want instead.

Be sure to set an appropriate Window filter, so this only affects the window your game is in.

----

BTW: Whenever you have a question (as opposed to an issue) it's better to post it on our [list](https://groups.google.com/forum/#!forum/autokey-users) or on [Gitter](https://gitter.im/autokey/autokey). The list is more user oriented whereas more technical questions will get a better answer on Gitter.

Both of these venues have a number of active users, so you are likely to get more, better, and faster answers on them than here on GitHub where only our very busy developers and a few users see the posts. We also prefer to have the developers developing rather than taking time to answer questions and manage issues that the user community could handle for them.

**Reply by HarmonBlues** [reply]
Thank you very much joseph. I'll use those avenues in the future.
